File: app/src/main/java/com/polije/sosrobahufactoryapp/ui/agen/home/peringkatStock/StockAgenRankFragment.java
package com.polije.sosrobahufactoryapp.ui.agen.home.peringkatStock;

import androidx.lifecycle.ViewModelProvider;

import android.os.Bundle;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;

import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.polije.sosrobahufactoryapp.R;

public class StockAgenRankFragment extends Fragment {

    private StockAgenRankViewModel mViewModel;

    public static StockAgenRankFragment newInstance() {
        return new StockAgenRankFragment();
    }

    @Override
    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container,
                             @Nullable Bundle savedInstanceState) {
        return inflater.inflate(R.layout.fragment_stock_agen_rank, container, false);
    }

    @Override
    public void onActivityCreated(@Nullable Bundle savedInstanceState) {
        super.onActivityCreated(savedInstanceState);
        mViewModel = new ViewModelProvider(this).get(StockAgenRankViewModel.class);
        // TODO: Use the ViewModel
    }

}

File: app/src/main/java/com/polije/sosrobahufactoryapp/ui/agen/home/peringkatStock/StockAgenRankFragment.kt
package com.polije.sosrobahufactoryapp.ui.agen.home.peringkatStock

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import androidx.lifecycle.ViewModelProvider
import com.polije.sosrobahufactoryapp.R

class StockAgenRankFragment : Fragment() {
    private var mViewModel: StockAgenRankViewModel? = null

    override fun onCreateView(
        inflater: LayoutInflater, container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {
        return inflater.inflate(R.layout.fragment_stock_agen_rank, container, false)
    }

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)

    }
    companion object {
        fun newInstance(): StockAgenRankFragment {
            return StockAgenRankFragment()
        }
    }
}